word_guess_art_class.rb

```diff
@@ -0,0 +1,43 @@
+class WordGuessArt
+
+  def initialize
+    @art_file = './turtle1.txt'
+  end
+
+  def print_art
+    File.open(@art_file) do |f|
+      f.each_line do |line|
+        puts line
+      end
+    end
+  end
+
+  #update_art method will rewrite the @art_file variable to the txt file we want or portion of text file we want
+  def update_art(max_wrong_guesses)
+    case max_wrong_guesses
```

Comment:
Would it be possible to generate the file name automatically from the number of wrong guesses, rather than having this big `case` statement?